plugins/modules/ec2_key.py

```diff
@@ -63,9 +73,11 @@
 EXAMPLES = r"""
 # Note: These examples do not set authentication details, see the AWS Guide for details.
 
-- name: create a new EC2 key pair, returns generated private key
+- name: create a new EC2 key pair, returns generated private key (use no_log to avoid private key being displayed into output)
```

Comment:
I would add this as a comment in the line below, like this:
```suggestion
- name: create a new EC2 key pair, returns generated private key
  # use no_log to avoid private key being displayed into output
```
A very long line will force many users to scroll when looking at the examples in the documentation.